client/src/renderer/components/Clipboard/index.jsx

```diff
@@ -0,0 +1,78 @@
+/** @format */
+
+/**
+ * Stores and lists contents of system clipboard
+ * @extends Component
+ */
+
+import "./styles.scss";
+import { Component, h } from 'preact';
+import clipboard from "electron-clipboard-extended";
+
+export default class Clipboard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {clipboardStack: []};
+    this.handleTextChanged = this.handleTextChanged.bind(this);
+    this.handleImageChanged = this.handleImageChanged.bind(this);
+  }
+
+  componentDidMount() {
+    clipboard.startWatching();
+    clipboard.on("text-changed", this.handleTextChanged);
+    clipboard.on("image-changed", this.handleImageChanged);
+  }
+
+  componentWillUnmount() {
+    clipboard.stopWatching();
+    clipboard.off("text-changed");
```

Comment:
Just noticed that you removed the `text-changed` event listener, but not the `image-changed` event listener when the component is unmounted.

```suggestion
    clipboard.off("text-changed");
    clipboard.off("image-changed");
```

Reply:
That probably explains why electron wasn't closing properly hahaha